stdlib/public/PointerBounds/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_swift_target_library(swift_PointerBounds ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_STDLIB
```

Comment:
We shouldn't create a new target here. The actual `macro` declaration and the types it depend on will go into the standard library so it's always available. In the short term, we can put them into the test cases themselves or (better) a module within the `test` directory that we build and feed in to the test command.

Reply:
Wouldn't placing it in the `test` dir rather than conditionally making it part of the stdlib make it harder to create pre-release versions for testing on real codebases?